enclave/core/sgx/calls.c

```diff
@@ -154,25 +155,22 @@ static oe_result_t _handle_init_enclave(uint64_t arg_in)
         if (_once == false)
         {
             /* Set the global enclave handle */
-            if (arg_in)
```

Comment:
Why we don't need to check whether `arg_in` is NULL anymore?

Reply:
@jxyang See my comment in line 145. oe_is_outside_enclave already checks for NULL parameters.